care/facility/migrations/0469_remove_scheduleexception_datetime_range_and_more.py

```
@@ -0,0 +1,61 @@
+    operations = [
+        migrations.RemoveField(
+            model_name='scheduleexception',
+            name='datetime_range',
+        ),
+        migrations.AddField(
+            model_name='availability',
+            name='reason',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='scheduleexception',
```

Ruff

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)